code/src/Dropdown.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react'
+
+export const Dropdown = ( {handleDropInput} ) => {
+    return <div className="question">
+      <label for="Movie" tabIndex="0">Rate Top Gun</label>
+        <select name="Movie" id="Movie" onChange={e => handleDropInput(e.target.value)} tabIndex="0" required>
```

Comment:
All of those `Movie` values should be written lowercase

Reply:
fixed